### DISCUSSION TITLE
Update bigdecimal dependency

### DISCUSSION THREAD

**Discussion by anghenfil** [diesel-rs/diesel#3073] [Ideas]
Hey,

it seems like diesel is using an quite old version of the bigdecimal crate (0.1.2) we should update to 0.3.0.

Best :)

**Comment by weiznich** [comment]
The master branch supports already bigdecimal 0.3.0. We plan to release that with our next feature release. Otherwise there are currently no plans to backport that to the 1.4.x release series and issue another patch release there.
